lighthouse-core/report/html/renderer/dom.js

```diff
@@ -77,6 +77,25 @@ class DOM {
     return element;
   }
 
+  /**
+   * @param {Document} document
+   */
+  removeCommentNodes(document) {
+    const walker = document.createTreeWalker(document, 128, null);
```

Comment:
128 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants). let's use `NodeFilter.SHOW_COMMENT`

Reply:
I did, but that fails in Node (during tests).

```
ReferenceError: NodeFilter is not defined
```

compromised w/ a const.